beacon-chain/state/state-native/v1/getters_attestation.go

```diff
@@ -6,50 +6,36 @@ import (
 
 // PreviousEpochAttestations corresponding to blocks on the beacon chain.
 func (b *BeaconState) PreviousEpochAttestations() ([]*ethpb.PendingAttestation, error) {
-	if !b.hasInnerState() {
-		return nil, nil
-	}
-	if b.state.PreviousEpochAttestations == nil {
+	if b.previousEpochAttestations == nil {
 		return nil, nil
 	}
 
 	b.lock.RLock()
 	defer b.lock.RUnlock()
 
-	return b.previousEpochAttestations(), nil
+	return b.previousEpochAttestationsInternal(), nil
 }
 
-// previousEpochAttestations corresponding to blocks on the beacon chain.
+// previousEpochAttestationsInternal corresponding to blocks on the beacon chain.
 // This assumes that a lock is already held on BeaconState.
-func (b *BeaconState) previousEpochAttestations() []*ethpb.PendingAttestation {
-	if !b.hasInnerState() {
-		return nil
-	}
-
-	return ethpb.CopyPendingAttestationSlice(b.state.PreviousEpochAttestations)
+func (b *BeaconState) previousEpochAttestationsInternal() []*ethpb.PendingAttestation {
```

Comment:
Function names have `internal` suffix because of collisions with state struct field names